examples/prometheus/prometheus.yaml

```diff
@@ -139,6 +147,58 @@ objects:
           - mountPath: /prometheus
             name: prometheus-data
 
+        # Deploy alertmanager behind prometheus-alert-buffer behind an oauth proxy
+        - name: alerts-oauth-proxy
+          image: ${IMAGE_PROXY}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 9443
+            name: web
+          args:
+          - -provider=openshift
+          - -https-address=:9443
+          - -http-address=:4190
```

Comment:
Oh.  I think this is a bug in oauth-proxy - it shouldn't listen on HTTP unless told.  Fine for now, can you open an oauth-proxy issue and assign to me?